src/Libraries/CoreNodesUI/Input/ISlider.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel;
+
+namespace DSCoreNodesUI.Input
+{
+    public interface ISlider<T>:INotifyPropertyChanged
```

Comment:
This looks pretty odd to me, I think I'm missing something in the design.

Why do we have the implementation of the type parameter on a generic interface, and have the sliders extend their data type (IntegerSlider : DSCoreNodesUI.Integer) rather than have the sliders extent a 'slider class' and parameterise the result?